src/main/java/ca/corefacility/bioinformatics/irida/model/workflow/analysis/AnalysisFastQC.java

```diff
@@ -464,7 +425,16 @@ public Long getTotalBases() {
 		return ImmutableSet.copyOf(overrepresentedSequences);
 	}
 
-	public AnalysisOutputFile getFastQCReport() {
```

Comment:
Is there a reason why you are removing the analysis fastqc report? I forget what it was supposed to contain. In our production database the field `fastQCReport_id` seems to be null. Should this be removed from our database as well?

Also, the private variable storing this report is still present in this class.

Reply:
Yeah as far as I can see it was never used anywhere.  I think at some point we were thinking of storing the full FastQC text report.  Prod db doesn't have it anywhere.  Dropping the column and removed it all from this file in 55e220441b.